server/data/data-api/src/test/java/org/apache/james/rrt/lib/CanSendFromContract.java

```diff
@@ -212,15 +215,24 @@ default void allValidFromAddressesShouldContainASendersAliasOfAnAliasOfTheUser()
     }
 
     @Test
-    default void allValidFromAddressesShouldNotContainASendersAliasOfAnAliasOfAnAliasOfTheUser() throws Exception {
-        Username userAliasBis = Username.of("aliasbis@" + DOMAIN.asString());
-        Username userAliasTer = Username.of("aliaster@" + DOMAIN.asString());
-        redirectUser(userAliasTer).to(userAliasBis);
-        redirectUser(userAliasBis).to(USER_ALIAS);
+    default void allValidFromAddressesShouldNotContainOutOfBoundsUserAliases() throws Exception {
```

Comment:
I don't understand this complicated test (not the name, and not the code)

Reply:
I take any alternative strategy

Reply:
I still dont understand this sorry.